### ISSUE TITLE
Page positioning

### ISSUE THREAD

**Issue by nickfenton** [blasten/turn.js#254]
When using a script based on the 'basic' sample, I assigned some pages to the class 'own-size'. These have re-sized fine, but when I open from the cover, the left hand page on the double spread appears in the top left, before moving back down to the center. Can this be fixed? Thanks.

**Comment by nickfenton** [comment]
Also a slightly different question. I am using turn.js for a blog. I am listing the blog articles which will link to the blog from a database. The php to get the blog details will be reliant on the query string, is there a way I can link to essentially refresh the page without the front cover coming back up? IE the double spread will stay open?

Many thanks

**Comment by blasten** [comment]
@nickfenton the _.own-size_ requires an explicit definition of the width and height of that page.   Please make sure that if you add this class you also have a `style="width:00px; height:00px"` attribute. 

Regarding your second question,  what about using Ajax?  jQuery has `$.get` http://api.jquery.com/jQuery.get/

**Comment by nickfenton** [comment]
Thanks very much - have sorted these issues.

Another thing I have noticed, again using something based on the basic sample, whenever I set a page transition to have the "hard" class the animation is a bit messy in IE9. Is this a bug, or is there a workaround for it? Just to elaborate on what I mean - when clicking on the front cover, the double page spread opens with hardly any animation, and it appears like the right hand page becomes visible before the left hand page does. Thanks,
